### PR TITLE
 backlog must-have features added (Rough draft)

### PR DESCRIPTION
**Pull request by minjupgeorge**
## General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [x] is linked to an issue (if it is related)
- [x] feedback is addressed (if any and if it is appropriate feedback.)

## Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work

### HTML

- [ ] pages besides `/index.html` are stored in `/pages`
- [ ] the code is well-formatted
- [ ] the HTML is valid
- [ ] there are no inline styles (example: `style='color: red;'`)
- [ ] there are no `<style>` tags with CSS, all styles are hrefs
- [ ] there is no inline JavaScript (example: `onclick='doSomething()'`)
- [ ] there are no `<script>` tags with JS, all JS is in an separate file
- [ ] `id`s are used for JavaScript only, not for CSS
- [ ] semantic tags are used
- [ ] spelling and grammar is correct in all site content

### CSS

- [ ] the code is well-formatted
- [ ] passes all the linting checks

### CODE REVIEW

File: planning/backlog.md [hunk 1]
```diff
@@ -9,15 +9,114 @@ create your own!
...
+
+  - [ ] _there is M & V, and impact section which describes the goals and values
+        of the organization (Job advisor)._
+
+### As a Job seeker
+
+- I would like to know some infomartaion about the job vacanies without
+  registration
+
+  - [ ] _there's is a search bar where user can search using keyword without
```
Comment (MMikhailova) [inline planning/backlog.md:40]: Are we going to publish job vacancies? 
To do it we have to enable companies to log in and publish vacancies. 
We are not eligible to create job vacancies by ourselves.

File: planning/backlog.md [hunk 2]
```diff
@@ -9,15 +9,114 @@ create your own!
...
+        registration._
+
+- I would like to know some infomartaion about companies which have Gender
+  equality
+
+  - [ ] _there's is a "Gender equality" button below search bar which contains
+        brief information about the vacancies offered for freshers_
```
Comment (MMikhailova) [inline planning/backlog.md:47]: Why only for freshers (candidate without job experience)? Experience job seekers might  also be interested in gender equality?

**Review by MMikhailova — COMMENTED** [review]
@minjupgeorge Please see my comments

File: planning/backlog.md (hunk 1 above)
Comment (minjupgeorge) [reply planning/backlog.md:40]: Thanks maria.. can we put this in to could-have.

File: planning/backlog.md (hunk 2 above)
Comment (minjupgeorge) [reply planning/backlog.md:47]: Then let me move it under jobseeker

**Review by MMikhailova — APPROVED** [review]

**Pull request merged**